README.md

```diff
@@ -55,13 +52,13 @@ Requirements:
 - You need `jdk` 11+ installed to compile (and `jre` 11 to run)
 - Anything else that `paper` requires to build
 
-Run the following commands in the root directory:
+If all you want it a paperclip server jar, just run `./tuinity jar`
 
-```
-./tuinity patch
-```
+Otherwise, to setup the `Tuinity-API` and `Tuinity-Server` repo, just run the following command
+in your project root `./tuinity patch` or you can run `./tuinity build` to build the 
```

Comment:
build does not run a patch op, only patch/jar do patch ops
the `or` implies the build will run a patch op